exetera/core/fields.py

```diff
@@ -633,24 +968,89 @@ def apply_index(self, index_to_apply, target=None, in_place=False):
+        """
         return FieldDataOps.apply_spans_last(self, spans_to_apply, target, in_place)
 
     def apply_spans_min(self, spans_to_apply, target=None, in_place=False):
+        """
+        Apply spans (min). This operation doesn't modify the field on which it
+        is called unless 'in_place is set to true'. The user can specify a 'target' field that
+        the reindexed data is written to.
+
+        :param index_to_apply: a Field or numpy array that contains the indices
+        :param target: if set, this is the field that is written to. This field must be writable.
+            If 'target' is set, 'in_place' must be False.
+        :param in_place: if True, perform the operation destructively on this field. This field
+            must be writable. If 'in_place' is True, 'target' must be None
+        :return: The respanned field. This is a new field instance unless 'target' is set, in which
+            case it is the target field, or unless 'in_place' is True, in which case it is this field.
+        """
         return FieldDataOps.apply_spans_min(self, spans_to_apply, target, in_place)
 
     def apply_spans_max(self, spans_to_apply, target=None, in_place=False):
+        """
+        Apply spans (max). This operation doesn't modify the field on which it
+        is called unless 'in_place is set to true'. The user can specify a 'target' field that
+        the reindexed data is written to.
+
+        :param index_to_apply: a Field or numpy array that contains the indices
+        :param target: if set, this is the field that is written to. This field must be writable.
+            If 'target' is set, 'in_place' must be False.
+        :param in_place: if True, perform the operation destructively on this field. This field
+            must be writable. If 'in_place' is True, 'target' must be None
+        :return: The respanned field. This is a new field instance unless 'target' is set, in which
+            case it is the target field, or unless 'in_place' is True, in which case it is this field.
+        """
         return FieldDataOps.apply_spans_max(self, spans_to_apply, target, in_place)
 
     def isin(self, test_elements:Union[list, set, np.ndarray]):
+        """
+        Returns a boolean array of the same length as field \
+        that is True where an element of field is in test_elements and False otherwise.
+
+        :param test_elements: The values against which to test each value of field.
+        :return: a boolean array of the same length as field
+        """
         return FieldDataOps.apply_isin(self, test_elements)
 
     def unique(self, return_index=False, return_inverse=False, return_counts=False):
-        "Find the unique elements of FixedStringMemField"
-        return FieldDataOps.apply_unique(self, return_index, return_inverse, return_counts)
+        """
+        Find the unique elements of FixedStringMemField
```

Comment:
Maybe a bit more elaboration on what unique gives you, an example, explanation of what the arguments do for the return value. Mention that it's similar to np.unique.